### PR TITLE
VIH-7953 logout script changes

### PR DESCRIPTION
**Pull request by ibthmcts**
logout script changes

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###
https://tools.hmcts.net/jira/browse/VIH-7953

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by MichaelKeaneathmcts — DISMISSED** [review]

File: ServiceWebsite/ServiceWebsite.AcceptanceTests/Steps/LoginSteps.cs [hunk 1]
```diff
@@ -44,6 +45,7 @@ public void WhenTheUserAttemptsToLogout()
         public void WhenTheUserSignsOut()
         {
             WhenTheUserAttemptsToLogout();
+            Thread.Sleep(5000);
```
Comment (PrasannaDommalapati) [inline ServiceWebsite/ServiceWebsite.AcceptanceTests/Steps/LoginSteps.cs:48]: Could you remove thread.sleep

**Review by PrasannaDommalapati — CHANGES_REQUESTED** [review]

File: ServiceWebsite/ServiceWebsite.AcceptanceTests/Steps/LoginSteps.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (ibthmcts) [reply ServiceWebsite/ServiceWebsite.AcceptanceTests/Steps/LoginSteps.cs]: I've modified to now use **_browsers[_c.CurrentUser].Driver.WaitUntilVisible(...)** to assert the sign out message, whenever it appears.

Comment (ibthmcts) [reply ServiceWebsite/ServiceWebsite.AcceptanceTests/Steps/LoginSteps.cs]: I've modified to now use **_browsers[_c.CurrentUser].Driver.WaitUntilVisible(...)** to assert the sign out message, whenever it appears.

**Review by PrasannaDommalapati — APPROVED** [review]
LGTM

**Pull request merged**